doc/sources/guide/packaging-osx.rst

```diff
@@ -1,82 +1,192 @@
 Creating packages for OS X
 ==========================
 
-Packaging your application for the OS X 10.6 platform can only be done inside OS X.
-
-The package will only work for the 64 bit OS X. We no longer support 32 bit OS X platforms.
-
 .. note::
-    Currently, packages for OS X can only be generated with Python 2.7. Python 3.3+ support is on the way...
+    - Packaging your application for the OS X platform can only be done inside OS X.
 
-.. _osx_requirements:
 
-Official Packaging method
--------------------------
+Using kivy-sdk-packager
+-----------------------
+.. note::
+    - This is currently the preferred way to package an app if you are new to Kivy
+      and/or you know that you do not need newer or more libraries than provided by the Kivy.app
+    - The package will only work for the 64 bit OS X. We no longer support 32 bit OS X platforms.
+    - Currently, packages built using this method can only be generated with Python 2.7.
+      Python 3.3+ support is on the way...
 
-Due to a lot of problems with including libraries and files on OS X with other methods
-we now have a simpler and easier way to package Kivy apps on OS X.
+Since Kivy 1.9 Kivy package on OS X is a self contained portable distribution.
+It is now possible to package Kivy apps using the method described below to make
+it easier to include frameworks like SDL 2 and GStreamer.
 
-Since kivy 1.9 kivy package on OS X is a self contained portable distribution.
-It is now possible to package kivy apps using the method described below to make
-it easier to include frameworks like sdl2 and gstreamer::
+1. Make sure you have the unmodified Kivy.app from the download page.
 
-1: Make sure you have the Kivy.app(unmodified) from the download page.
-2: run the following commands::
+2. Run the following commands::
 
-    > mkdir  packaging
+    > mkdir packaging
     > cd packaging
     packaging> git clone https://github.com/kivy/kivy-sdk-packager
     packaging> cd kivy-sdk-packager/osx
     osx> cp -a /Applications/Kivy.app ./Kivy.App
 
-This step above is important, you have to make sure to preserve the paths and permissions. A command like cp -rf will copy but make the app unusable and lead to error later on.
+  .. note::
+    - This step above is important, you have to make sure to preserve the paths and permissions.
+      A command like ``cp -rf`` will copy but make the app unusable and lead to error later on.
 
-Now all you need to do is to include your compiled app into the Kivy.app, simply run the following command::
+3. Now all you need to do is to include your compiled app into the Kivy.app. Simply run the following command::
 
-    osx> ./package-app.sh path/to/your/app
+    osx> ./package-app.sh /Applications/Kivy.app/Contents/Resources/kivy/examples/demo/touchtracer/
```

Comment:
Packaging touchtracer is what we document for every other OS too. And I think it is quite nice to have an example everyone uses.